packages/hoppscotch-ui/src/components/smart/Modal.vue

```diff
@@ -154,7 +162,7 @@ onMounted(() => {
   stack.push(stackId.value)
   document.addEventListener("keydown", onKeyDown)
 
-  // disableKeybindings()
+  onModalOpen && onModalOpen()
```

Comment:
See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#optional_chaining_with_function_calls

```suggestion
  onModalOpen?.()
```

Reply:
Thanks. I'll remember this from the next time :D